### PR TITLE
Make ESLint search within subfolders & fix ESLint errors

### PR DESCRIPTION
**Pull request by techncl**
When we were running 'npm run lint', it wasn't searching the subfolder of 'upload', so this should fix that

### CODE REVIEW

File: npm/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "build-css": "npm-run-all add-stylesheet-dir sass-compile compress-css",
     "build:local": "npm-run-all copy-assets build-css build-ts:prod",
     "build": "npm-run-all copy-assets build-css build-ts:prod",
-    "lint": "eslint src/**/*.ts",
+    "lint": "eslint src/**/*.ts src/**/**/*.ts",
```
Comment (vimleshtna) [inline npm/package.json:17]: I think if you use just a single quote then it works recursively.

`"lint": "eslint 'src/**/*.ts'"`

**Review by vimleshtna — COMMENTED** [review]
Looks good, just a suggestion, it should work.

File: npm/package.json (hunk 1 above)
Comment (techncl) [reply npm/package.json:17]: Oh, cool. Thanks.

**Review by vimleshtna — APPROVED** [review]
Looks good, thanks.

**Pull request merged**